examples/diamond/BUILD

```diff
@@ -29,5 +29,10 @@ csharp_library(
 csharp_library(
     name = "Bottom",
     srcs = ["Bottom.cs"],
-    target_frameworks= ["net48", "net40", "netstandard1.4"],
+    target_frameworks= [
+        "net48",
+        "net40",
+        # TODO: change this back to "netstandard1.4" when we support it
```

Comment:
The new reference to `@net//:mscorlib` broke this example, naturally.

Reply:
Ah, I realized two things:

* I should add a test for not including the stdlib
* For now it'd be reasonable to make diamond be that test and keep it reference netstandard1.4 (which is what it does in the docs).

Doing that I discovered that `stdlib = None` wouldn't actually work. So `stdlib = False` is how you control it, and there is a private default attr `_stdlib` with the target.

If you want to "provide your own standard library" you'd add it to `deps` like normal.